docs/tutorials/services/color-detection-scuttle.md

```diff
@@ -16,8 +16,8 @@ authors: [ "Hazal Mestci" ]
 languages: [ "python" ]
 viamresources: [ "base", "vision", "camera" ]
 level: "Beginner"
-date: "18 August 2022"
-updated: "11 August 2023"
+date: "2022-08-18"
```

Comment:
(opt) I don't want to block bc I don't think it matters too much but I will say american ppl might get confused by this date format 
```suggestion
date: "08-18-2022"
```

Reply:
I think it's not just american people, but in this case this is how Go expects the date to be formatted, so it is what it is